src/webapp/js/components/sidebar.js

```diff
@@ -318,6 +318,17 @@ class SideBar extends Component {
             </Link>
           </li>
         ))}
+        <li key="automated-reports" className="border-none">
```

Comment:
I don't think this should be in the reports submenu, it can be below the reports in the main menu